programs/uxd/src/instructions/credix_lp/collect_profit_of_credix_lp_depository.rs

```diff
@@ -112,32 +109,36 @@ pub struct CollectProfitOfCredixLpDepository<'info> {
     )]
     pub credix_pass: Account<'info, credix_client::CredixPass>,
 
-    /// #14
+    /// #13
     #[account(
         mut,
         token::mint = collateral_mint,
     )]
     pub credix_treasury_collateral: Box<Account<'info, TokenAccount>>,
 
-    /// #15
+    /// #14
     /// CHECK: not used by us, checked by credix program
     pub credix_multisig_key: AccountInfo<'info>,
 
-    /// #16
+    /// #15
     #[account(
         mut,
         token::authority = credix_multisig_key,
         token::mint = collateral_mint,
     )]
     pub credix_multisig_collateral: Box<Account<'info, TokenAccount>>,
 
+    /// #16
+    /// CHECK: checked by the depository constraints, content not used
+    pub profits_beneficiary_key: AccountInfo<'info>,
```

Comment:
I think the lazier version is we just set the `profits_beneficiary_collateral` directly on the depository, then could just skip the owner checking

Reply:
but anyway if we decide to keep this, could just name it to `profits_beneficiary`, just like the `authority` won't be named as `authority_key`

Reply:
Yeah that make sense, ok i'll just remove this entierly more simple indeed! thanks king